useIsShake.js

```diff
@@ -1,59 +1,44 @@
 import React, { useState, useEffect } from 'react';
 import { Accelerometer } from 'expo-sensors';
-import { QUOTES_IN_DURATION, QUOTES_IN_DELAY, QUOTES_OUT_DURATION, SVG_LOAD_DURATION, SVG_OUT_DURATION, SVG_IN_DURATION, SVG_IN_DELAY, TRANSITION_TIME_2 } from './constants';
 
 export function useIsShake() {
   const [{ x }, setData] = useState({
     x: 0,
   });
   const [subscription, setSubscription] = useState(null);
-  const [toggle, setToggle] = useState(false);
+  const [isShakeTriggered, setIsShakeTriggered] = useState(false);
+  const [isToggleReady, setIsToggleReady] = useState(false);
 
   let polls = [null, null];
   let diffs = [];
-  let shakeCount = 0;
-  let isToggleReady;
-  let hasToggledBefore;
 
   function isShake({ x: newX }) {
-    if (isToggleReady === true) {
-      if (diffs.length === 2) {
-        shakeCount++;
```

Comment:
ShakeCount only relevant to development testing, not app functionality. No longer relevant that debounce functionality has been redesigned through listener removal and addition dependent up isToggleReady state.